Unity-Technologies/Graphics
# Close popup window when focus is lost (case 1258571)

### Purpose of this PR
Fixes [case 1258571 ](https://fogbugz.unity3d.com/f/cases/1258571/).

Volume component search popup did not close when focus is lost.

So I added a `Close` call in the `OnLostFocus` Unity method.

---
### Testing status

**Manual Tests**: 
1. On any HDRP project
1. Add a volume component to a game object
1. Click the "Add Override" button in the inspector of the volume component
1. Click on the popup to make sure it has all focus
1. Click somewhere else in the editor
1. > Observe the the popup is closed

---
### Comments to reviewers
